core/src/main/java/com/statful/client/core/api/MetricsSenderAPI.java

```diff
@@ -107,38 +204,23 @@ public SenderAPI aggregations(final Aggregation... aggregations) {
     }
 
     @Override
-    public SenderAPI aggregations(final Aggregations aggregations) {
+    public MetricsSenderAPI aggregations(final Aggregations aggregations) {
```

Comment:
This is a leftover of the initial implementation, it should return the interface and not the implementation. Good catch.